newm-server/src/main/kotlin/io/newm/server/features/song/database/SongEntity.kt

```diff
@@ -149,7 +147,6 @@ class SongEntity(
 
         fun genres(filters: SongFilters): SizedIterable<String> {
             val ops = filters.toOps()
-            val genre = SongTable.genres.unnest()
```

Comment:
I couldn't get our `unnest()` to work with the new Array support in exposed. I took the approach of just pulling all the data down and manipulating it in the kotlin code.